src/components/select/CdrSelect.vue

```diff
@@ -60,7 +60,7 @@ const props = defineProps({
   errorRole: {
     type: String,
     required: false,
-    default: 'status',
+    default: '',
```

Comment:
Removed this as a default because the error got announced twice

Reply:
Should the default be `undefined` then instead of an empty string?

Reply:
taking a look

Reply:
> Should the default be `undefined` then instead of an empty string?

Updated to remove the default property.